pkg/internal/client/client.go

```diff
@@ -70,6 +70,22 @@ func (c *Client) CreateVirtualMachineInstance(ctx context.Context,
 	}
 }
 
+func (c *Client) GetVirtualMachineInstance(ctx context.Context, namespace, name string) (*kvcorev1.VirtualMachineInstance, error) {
```

Comment:
I think the last 2 commits should be before the commit that inserts the actual delete..

Reply:
Could you please elaborate?

Reply:
ah I missed the last commit, please desregard!